alpakaConfig.cmake

```diff
@@ -395,19 +400,19 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                 ELSEIF("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
                     IF(CUDA_VERSION VERSION_EQUAL 8.0)
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 4.0)
-                            MESSAGE(FATAL_ERROR "NVCC 8.0 does not support clang 4+. Please use clang 3.9!")
+                            MESSAGE(FATAL_ERROR "NVCC 8.0 does not support clang 4+. Please use NVCC 9.1!")
                         ENDIF()
                     ELSEIF(CUDA_VERSION VERSION_EQUAL 9.0)
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 4.0)
-                            MESSAGE(FATAL_ERROR "NVCC 9.0 does not support clang 4+. Please use clang 3.9!")
+                            MESSAGE(FATAL_ERROR "NVCC 9.0 does not support clang 4+. Please use NVCC 9.1!")
                         ENDIF()
                     ELSEIF(CUDA_VERSION VERSION_EQUAL 9.1)
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.0)
                             MESSAGE(FATAL_ERROR "NVCC 9.1 does not support clang 5+. Please use clang 4!")
                         ENDIF()
                     ELSEIF(CUDA_VERSION VERSION_EQUAL 9.2)
                         IF(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 6.0)
-                            MESSAGE(FATAL_ERROR "NVCC 9.2 does not support clang 6+. Please use clang 4 or 5!")
+                            MESSAGE(FATAL_ERROR "NVCC 9.2 in combination with clang is not yet supported by alpaka!")
```

Comment:
I just saw that we already support CUDA 9.2 when using clang 4.0. Clang 5.0 seems to be a problem. So this should be changed to `NVCC 9.2 does not support clang 6+ and fails compiling with clang 5. Please use clang 4!`